site/content/docs/latest/howto/offline-installation.md

```diff
@@ -1,64 +1,100 @@
+
+## Step 1. Download the Kubeapps chart
 
-First, download the tarball containing the Kubeapps chart from the publicly available repository maintained by Bitnami. Note that Internet connection is necessary at this point:
+First, download the tarball containing the Kubeapps chart from the publicly available repository maintained by Bitnami.
 
 ```bash
-helm pull --untar https://charts.bitnami.com/bitnami/kubeapps-4.0.4.tgz
+helm pull --untar https://charts.bitnami.com/bitnami/kubeapps-x.y.z.tgz
 helm dep update ./kubeapps
 ```
 
-> Latest version of this chart available at the Bitnami Chart [Repository](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/Chart.yaml#L3)
+> Notice `x.y.z` must be replaced by the latest version of the Kubeapps chart available at the [Bitnami Chart Repository](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/Chart.yaml#L3)
 
-## 2. Mirror Kubeapps images
+## Step 2. Mirror Kubeapps images
 
-In order to be able to install Kubeapps, it's necessary to either have a copy of all the images that Kubeapps requires in each node of the cluster or push these images to an internal Docker registry that Kubernetes can access. You can obtain the list of images by checking the `values.yaml` of the chart. For example:
+To be able to install Kubeapps, it's necessary to either have a copy of all the images that Kubeapps requires in each node of the cluster or push these images to an internal Docker registry that Kubernetes can access. You can obtain the list of images by checking the `values.yaml` of the chart. For example:
 
 ```yaml
 registry: docker.io
 repository: bitnami/nginx
 tag: 1.19.2-debian-10-r32
 ```
 
-> This list includes but is not limited to: `bitnami/kubeapps-apprepository-controller`, `bitnami/kubeapps-asset-syncer`, `bitnami/kubeapps-dashboard`, `bitnami/kubeapps-pinniped-proxy`, `bitnami/kubeapps-apis`, `bitnami/nginx`, `bitnami/oauth2-proxy`, `bitnami/postgresql`.
+> The list of images to download includes (could change in time):
+>
+> - `bitnami/nginx`
+> - `bitnami/kubeapps-dashboard`
+> - `bitnami/kubeapps-apprepository-controller`
+> - `bitnami/kubeapps-asset-syncer`
+> - `bitnami/oauth2-proxy`
+> - `bitnami/kubeapps-pinniped-proxy`
+> - `bitnami/kubeapps-apis`
+> - `bitnami/postgresql`
 
-For simplicity, in this guide, we use a single-node cluster created with [Kubernetes in Docker (`kind`)](https://github.com/kubernetes-sigs/kind). In this environment, as the images have to be preloaded, we first have to pull the images (`docker pull`) and next load them into the cluster (`kind load docker-image`):
+For simplicity, in this guide, use a single-node cluster created with [Kubernetes in Docker (`kind`)](https://github.com/kubernetes-sigs/kind) with a namespace called "kubeapps". In this environment, the images have to be preloaded: - first, pull the images (`docker pull`), - next load them into the cluster (`kind load docker-image`).
```

Comment:
In the rest of the document you're using `pre-xxxxx`, so changing it for consistency:

```suggestion
For simplicity, in this guide, use a single-node cluster created with [Kubernetes in Docker (`kind`)](https://github.com/kubernetes-sigs/kind), using a namespace called "kubeapps". In this environment, the images have to be pre-loaded: - first, pull the images (`docker pull`), - next load them into the cluster (`kind load docker-image`).
```